src/app/capture_thread.cpp

```diff
@@ -106,10 +117,14 @@ void CaptureThread::selectCaptureMethod() {
     new_capture = captureFiles;
   } else if(captureModule->getString() == "Generator") {
     new_capture = captureGenerator;
-  } else if(captureModule->getString() == "DC 1394") {
-    new_capture = captureDC1394;
-  } else {
+#ifdef MVIMPACT
+  } else if(captureModule->getString() == "BlueFox2") {
+    new_capture = captureBlueFox2;
+#endif
+  } else if(captureModule->getString() == "Video 4 Linux") {
     new_capture = captureV4L;
+  } else {
+    new_capture = captureDC1394;
```

Comment:
With the addition of more drivers, we should explicitly check for all strings, rather than have a final default. 
It'll help avoid future potential bugs from being introduced while adding new capture drivers.
Could you please update to add an explicit string check?